test/recipes/30-test_evp_data/evppbe_pkcs12.txt

```diff
@@ -13,6 +13,7 @@
 
 Title = PKCS12 tests
 
+Availablein = default
```

Comment:
If this entire file is only useful for the default case then maybe just more the entry for this file in 30-test_evp.t instead of doing this.

Reply:
I'm not sure if we'll need this in the FIPS/base provider or not at this point.  Leaving this hedges our bets.

Reply:
I've moved it, it can always be put back if we need to test it in FIPS.

Reply:
Cheers - otherwise it starts spreading everywhere.. I removed most of them from all files.